deployment/helm/node-feature-discovery/templates/master.yaml

```
@@ -78,6 +78,9 @@ spec:
             {{- if .Values.master.instance | empty | not }}
             - "-instance={{ .Values.master.instance }}"
             {{- end }}
+            {{- if .Values.master.port | empty | not }}
+            - "-port={{ .Values.master.port }}"
+            {{- end }}
```

nit: I think we could use the same simper construct than with the other port-related changes you did. On a sidenote this also makes things work in the very-very unlikely scenario that we would change the default for the `-port` flag in the nfd-master daemon 😅

```suggestion
            - "-port={{ .Values.master.port | default "8080" }}"
```